DataFormats/HcalRecHit/BuildFile.xml

```diff
@@ -2,6 +2,12 @@
 <use name="DataFormats/Common"/>
 <use name="DataFormats/HcalDetId"/>
 <use name="DataFormats/HcalDigi"/>
+<use name="DataFormats/DetId"/>
+<use name="DataFormats/Portable"/>
+<use name="DataFormats/SoATemplate"/>
+<use name="FWCore/MessageLogger"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<flags ALPAKA_BACKENDS="cuda"/>
```

Comment:
Why only `cuda` ?

Reply:
Changed to `!serial` in 4615e8825352834ac1dd4f1bbe766b0cafc1d321